src/main/java/training/chessington/model/pieces/Pawn.java

```diff
@@ -6,15 +6,108 @@
 import training.chessington.model.PlayerColour;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 
+
 public class Pawn extends AbstractPiece {
+    private int direction;
     public Pawn(PlayerColour colour) {
         super(Piece.PieceType.PAWN, colour);
+        this.direction = (this.getColour() == PlayerColour.WHITE ? -1: 1);
     }
 
     @Override
     public List<Move> getAllowedMoves(Coordinates from, Board board) {
-        return new ArrayList<>();
+
+        List<Move> allowableMoves = new ArrayList<>();
+
+        // check if at end of boundary TOP and BOTTOM
+        boolean atEndOfBoard = boundaryCheck(from.getRow());
+        if (atEndOfBoard) { return allowableMoves; }
+
+        // check for nearby enemies to capture
+        captureNearbyEnemy(from, board, allowableMoves);
+
+        // check for obstruction directly in front
+        boolean pieceInfrontExists = pieceInfront(from, board);
+        if (pieceInfrontExists) { return allowableMoves; }
+
+
+        //  ########     standard move one place forward
+
+        // make the new coordinate object
+        Coordinates to = from.plus(direction, 0);
+        // add move object to list of move objects
+        allowableMoves.add(new Move(from, to));
+
+        //  ########      end of standard move one place forward
+
+        // check if it can move two spaces
+        hasItMovedAtAll(from, allowableMoves, board);
+
+        return allowableMoves;
     }
+
+    public void hasItMovedAtAll(Coordinates from, List<Move> initAllowableMoves, Board board) {
+
+        int initRowBlack = 1;
+        int initRowWhite = 6;

```

Comment:
you could have this be just called `initalRow` and assign it a value based on the player colour. Then you don't need the OR in your next if statement. It can just be `if fromRow == initialRow`